src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/Index.cshtml

```diff
@@ -103,15 +103,25 @@ else
                         </div>
                     }
 
-                    @if (Model.Data.ApprenticeshipEmployerType == ApprenticeshipEmployerType.Levy 
-                         && Html.IsAuthorized("EmployerFeature.TransfersMatching"))
+                    @if (Html.IsAuthorized("EmployerFeature.TransfersMatching"))
                     {
-                        <div class="card">
-                            <h3 class="heading-medium"><a href="@Url.EmployerFinanceAction("transfers")">Your transfers</a></h3>
-                            <p>View your 25% transfers balance and manage your transfers, including finding a business to transfer your levy to.</p>
-                        </div>
+                        if (Model.Data.ApprenticeshipEmployerType == ApprenticeshipEmployerType.Levy)
```

Comment:
The url is the same is that correct? other than that looks good

Reply:
Yep this url should be the same